data/get_started.yml

```diff
@@ -1,187 +1,89 @@
-about:
+qcportal:
   enable: true
-  title : "View data with QCPortal"
+  title : "Explore data with QCPortal"
   content : "
-QCPortal is the Python front-end for analyzing and visualizing results.
-QCPortal contains many types of <i>Collections</i> that organize large amounts
-of data in canonical ways such as a <i>ReactionDataset</i> which contains many
-common interaction energy, thermochemical, and conformational datasets from the
-literature.
+QCPortal is the primary front-end for accessing, analyzing, and visualizing QCArchive data.
+QCPortal provides models that organize large sets of calculations into structured Python objects.
+Get started with our examples.
+<br><br>
 
-<br></br>
-
-Get started with our examples:
-
-<div class='col-12 text-left'>
-        <a href='http://docs.qcarchive.molssi.org/en/latest/' class='btn btn-main-md'>See Examples</a>
+<div class='col-12 text-center'>
+        <a href='/examples' class='btn btn-main-md'>See Examples</a>
 </div>
-
   "
   imageSlider:
     - image : "images/about/client_connection.png"
     - image : "images/about/violin_plot.png"
     - image : "images/about/torsiondrive_plot.png"
 
-about2:
-  enable: false
-  title : "Execute Complex Workflows"
-  content : "
-This is another slider aligned to the right
-
-<br></br>
-Get started with our examples:
-
-<div class='col-12 text-left'>
-        <a href='mailto:qcarchive@molssi.org' class='btn btn-main-md'>Contact Us</a>
-</div>
-
+compute:
+    enable: true
+    title : "Compute with QCArchive"
+    image : "images/about/openff_workflow.png"
+    content : "
+  QCArchive provides a complete platform for computing, managing, and organizing large-scale quantum chemical workflows and datasets.
+  This platform is already used by the OpenFF project, and we are actively looking
```

Comment:
"This platform is already used by many partner projects"

I don't fully get the "we can run your compute queue" for you from this sentence. Maybe more like:
"The platform is already by many partner groups to aggregate open data and distribute it to the community. We are actively looking to work with additional partners to generate additional open data within QCArchive for machine learning, force field prediction, methodology assessment, and more."